src/vivarium_cluster_tools/psimulate/worker/load_test_work_horse.py

```diff
@@ -21,16 +21,42 @@
 LOAD_TEST_WORK_HORSE_IMPORT_PATH = f"{__name__}.work_horse"
 
 
+def get_psimulate_test_dict():
+    return {
+        "sleep": sleep_test,
+        "large_results": large_results_test
+    }
+
+
+def sleep_test(job_parameters: JobParameters) -> pd.DataFrame:
+    min_sleep_time = 5
+    max_sleep_time = 60
+    rs = np.random.RandomState(seed=get_hash(f"sleep_test_{job_parameters.random_seed}"))
+    sleep_time = rs.randint(min_sleep_time, max_sleep_time)
+
+    logger.info(f"Sleeping for {sleep_time}s.")
+    time.sleep(sleep_time)
```

Comment:
is this mimicking something that was happening in the production runs where you were getting connection errors, or does this serve another function?